cinema/serializers.py

```diff
@@ -1,23 +1,89 @@
+from enum import unique
+
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Genre, Actor, CinemaHall
 
 
 class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` for `MovieSerializer` to simplify the code. This will automatically generate fields based on the model definition and reduce redundancy.